### DISCUSSION TITLE
No signal from ebus

### DISCUSSION THREAD

**Discussion by iamjlm** [john30/ebusd#868] [Q&A]
Hello,

I have recently installed the ebus adapter 3.1. It is the one with a wifi interface (wemos D1 mini).

But it is not working …. I would need you help ! Thanks in advance.

The "ebusd" is installed on a Raspberry PI4 and seems to work as expected. In the ebusd log I have :

> 2023-03-30 09:25:39.407 [main notice] ebusd 23.1.23.1 started with auto scan on enhanced device 10.66.0.248:9999 
> 2023-03-30 09:25:41.388 [bus notice] device status: resetting 
> 2023-03-30 09:25:41.388 [bus notice] bus started with own address 31/36 
> 2023-03-30 09:25:41.390 [mqtt notice] connection established_

This is confirmed by the « ebus adapter 3 » status :

_> [eBUS adapter 3](http://10.66.0.248/)
> Build: 20221215
> ebusd device string: ens:10.66.0.248:9999
> WiFi signal: 92% (-54dBm)
> OTA enabled: waiting
>     • [Status](http://10.66.0.248/)running 
>       
> Chip ID: 00f7dfd6, [d1mini@80](http://10.66.0.248/?verbose=)
> Hostname: ebus-f7dfd6
> Up time: 26839
> Free heap: 24976
> ebusd connected: yes (inactive)
> eBUS signal: no signal_ 

The trouble is coming from the eBus connection : « no signal ».
The, yellow led on and  green led of the adapter is flashing quickly as soon as I connect the ebus to the interface, showing some "life" on the bus
The connection are made on the red connector on my Vaillant heating system.

My Vaillant equipments are the following :
- Vaillant « auroCompact/2 » heating system,
- wireless control module (VRC-470f) composed of a wireless receiver installed on the heating system front panel (and connected to the ebus), and a wireless control interface communicating with the receiver in wireless.

I made some test with the Vaillant heater. In particular I tested if the ebus connector, so to make sure it is working correctly.
I removed the wireless receiver from the heating system, and  connected it with wires directly to the above red connector. I checked that in this wiring configuration that the wireless interface was still working. Worked fine.

Once confirmed, I connected the "ebus adapter 3.1" in parallel to the wireless receiver, knowing that the ebus « red » connector is working.

Still « no signal » status on the main page, while yellow led is on, and green led flashing 
ebus adapter log entry :
_

> 17:31:47.4: wifi evt 8
> 17:31:49.9: wifi evt 2
> 17:31:52.4: wifi evt 0
> 17:31:53.9: wifi evt 3
> 17:31:56.0: check eBUS conn
> 17:31:56.3: check eBUS conn: 0
> 17:31:56.3: eBUS >SYN enh
> 17:31:56.3: skip
> 17:31:56.3: skip first 1
> 17:31:56.3: setup: error=0, wifi=1, ip=1
> 17:32:01.3: start
> 17:32:04.2: conn 10.66.0.26:39800

_
As I have a few wemos D1 available, I took a new one and flashed the «latest ebus-v3_d1mini.bin firmware on it.
Still no sucess.

Could you help me ? Is there something wrong in my configuration ?
_

> Build: 20221215
> ebusd device string: ens:10.66.0.248:9999
> WiFi signal: 100% (-49dBm)
>     • WIFI SSID:  xxxxxx
>     • WIFI secret: xxxxxx
>     • WIFI IP address: DHCP
> WIFI power: normal (17 dbm)
>     • WIFI phy mode: 
>     • WIFI hostname: ebus-f7dfd6
>     • eBUS RX+TX PINs: ebusd connection: Adapter 3.1 RX+TX high speed (if enhanced)=GPIO3+1
>     • on port: 9999
>     • HTTP TCP port: 80 (0 to disable) 
>     • LED PINs: RX: disabled , TX: disabled
>     • [Initial PINs](http://10.66.0.248/pins)

_

**Comment by Juergen2020** [comment]
did you get it running by now?
